docs/changelog/0037-upgrade-to-elasticsearch7.md

```diff
@@ -0,0 +1,357 @@
+<!--THIS FILE IS AUTOGENERATED: DO NOT EDIT-->
+<!--See https://github.com/dimagi/commcare-cloud/blob/master/changelog/README.md for instructions-->
+# 37. ES upgrade from 2.4.6 to 7.9.1
+
+**Date:** 2020-10-01
+
+**Optional per env:** _required on all environments_
+
+
+## CommCare Version Dependency
+The following version of CommCare must be deployed before rolling out this change:
+[9bd24b8f](https://github.com/dimagi/commcare-hq/commit/9bd24b8ff26266e65be2f093796b64116da72dac)
+
+
+## Change Context
+This change upgrades Elasticsearch from 2.4.6 to 7.9.1 version.
+CommCare HQ releases after **Todo; decide** will not continue to support Elasticsearch 2.4.6,
```

Comment:
TODO

Reply:
Yes! Overall timelines are still not decided.